recipes/botan/all/conanfile.py

```diff
@@ -283,7 +283,7 @@ def _configure_cmd(self):
 
         if self.options.with_openssl:
             build_flags.append('--with-openssl')
-            build_flags.extend(self._dependency_build_flags("OpenSSL"))
+            build_flags.extend(self._dependency_build_flags("openssl"))
```

Comment:
This is used as a name to find dependency flags.
OpenSSL is the wrong name, I compiled with -o botan:with_openssl=True and it works well on my machine after this change (tm).